WordPress/Classes/ViewRelated/Blog/My Site/MySiteViewController.swift

```diff
@@ -557,6 +557,14 @@ class MySiteViewController: UIViewController, NoResultsViewHost {
         let sitePickerViewController = SitePickerViewController(blog: blog, meScenePresenter: meScenePresenter)
 
         sitePickerViewController.onBlogSwitched = { [weak self] blog in
+
+            let isShowingDashboard = self?.segmentedControl.selectedSegmentIndex == Section.dashboard.rawValue
```

Comment:
Nit: we can extract `isShowingDashboard` to its own var (it's probably going to be useful in the future).